docs/developer/order/order-update.mdx

```diff
@@ -0,0 +1,93 @@
+
+### Available Fields
+
+- `billingAddress` – Update the customer's billing address
+- `shippingAddress` – Update the customer's shipping address
+- `userEmail` – Correct or change the customer's contact email
+- `externalReference` – Add or update an external system ID for integrations
+- `metadata` / `privateMetadata` – Store public or internal custom metadata for business logic or integrations
+- `languageCode` – Set or change the language code for localized communications
+
+### Example Use Cases
+
+- Correcting address typos reported by the customer
+- Updating customer email for notification delivery
+- Tagging the order with an ERP reference ID
+- Storing custom status or fraud-check flags in private metadata
+- Changing the language of invoice or email templates
+
+## Updating Unconfirmed Orders
+
+For UNCONFIRMED orders, more substantial edits are allowed. These include modifying discounts, lines, and other aspects of the order.
+
+### Managing Order Lines
+
+- [`orderLinesCreate`](/api-reference/orders/mutations/order-lines-create) – Add new product lines to the order
+- [`orderLineUpdate`](/api-reference/orders/mutations/order-line-update) – Change the quantity or variant of an existing order line
+- [`orderLineDelete`](/api-reference/orders/mutations/order-line-delete) – Remove a product line from the order
+
+### Managing Shipping Method
+
+- [`orderUpdateShipping`](/api-reference/orders/mutations/order-update-shipping) – Change the shipping method for an unconfirmed order
+
+### Managing Discounts 
+
+For more information about adding manual discounts, see the [Manual Discounts](/developer/discounts/manual-discounts) documentation.
+
+#### Order Level Discounts
+
+- [`orderDiscountAdd`](/api-reference/orders/mutations/order-discount-add) – Add a discount to the entire order
+- [`orderDiscountUpdate`](/api-reference/orders/mutations/order-discount-update) – Update an existing order-level discount
+- [`orderDiscountDelete`](/api-reference/orders/mutations/order-discount-delete) – Remove a discount from the order
+
+#### Order Line Level Discounts
+
+- [`orderLineDiscountUpdate`](/api-reference/orders/mutations/order-line-discount-update) – Apply or modify a discount on a specific order line
+- [`orderLineDiscountRemove`](/api-reference/orders/mutations/order-line-discount-remove) – Remove a line-level discount
+
+### Other Changes
+
+The [`orderUpdate`](/api-reference/orders/mutations/order-update) mutation can still be used for unconfirmed orders to change:
+- Customer email
+- Billing and shipping addresses
+- Order metadata
+
+## Order Updates Summary
+
+| Order State | What Can Be Modified |
+|--------------|----------------------|
+| Orders in `UNCONFIRMED` status | Lines, line discounts, order-level discounts, shipping method, addresses, email, metadata | 
+| Confirmed orders (`UNFULFILLED`, `FULFILLED`, `RETURNED`) | Addresses, email, external references, language, metadata |
```

Comment:
Suggestion: Maybe `Confirmed orders in UNFULFILLED, FULFILLED, or RETURNED status` to match the style from the first row?